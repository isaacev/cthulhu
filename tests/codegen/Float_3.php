<?php

namespace Float_3 {
  function main() {
    $x = 2.0 * 2.1;
  }
}

namespace {
  \Float_3\main(null);
}
