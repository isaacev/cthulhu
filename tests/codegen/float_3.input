#[entry]
fn main() -> () {
  let x: ::Kernel::Float = 2.0 * 2.1;
}
