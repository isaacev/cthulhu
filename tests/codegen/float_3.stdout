<?php

namespace float_3 {
  // #[entry]
  function main() {
    $x = 2.0 * 2.1;
  }
}

namespace {
  \float_3\main();
}
